packages/app-mobile/components/screens/folder.js

```diff
@@ -94,7 +100,20 @@ class FolderScreenComponent extends BaseScreenComponent {
 					autoFocus={true}
 					value={this.state.folder.title}
 					onChangeText={text => this.title_changeText(text)}
+					disabled={this.state.folder.encryption_applied}
 				/>
+				<View style={styles.folderPickerContainer}>
+					<FolderPicker
+						themeId={this.props.themeId}
+						placeholder={_('Select parent notebook')}
```

Comment:
When I choose the "Select parent notebook" option for an existing notebook, I get this:
![Could not be saved: SQLite not null exception](https://user-images.githubusercontent.com/46334387/233100312-96fd76db-9aed-4cd3-97ee-ed3a5b93f191.png)

Reply:
Ah seems to be due to the "Select parent notebook" having the value "null", seems they need to be an empty string for no parent. Also noticed you can set a parent notebook to be parented to a nested notebook, so I copied over the check for that from the desktop version including the message "Cannot move notebook to this location".